src/views/orb-map-view.ts

```diff
@@ -82,14 +85,18 @@ export class OrbMapView<N extends INodeBase, E extends IEdgeBase> implements IOr
 
   constructor(container: HTMLElement, settings: IOrbMapViewSettingsInit<N, E>) {
     this._container = container;
-    this._graph = new Graph<N, E>(undefined, {
-      onLoadedImages: () => {
-        // Not to call render() before user's .render()
-        if (this._renderer.isInitiallyRendered) {
-          this.render();
-        }
+    this._graph = new Graph<N, E>(
+      undefined,
+      {
+        onLoadedImages: () => {
+          // Not to call render() before user's .render()
+          if (this._renderer.isInitiallyRendered) {
+            this.render();
+          }
+        },
       },
-    });
+      this,
```

Comment:
What I was saying in one of the comments before :D This is hard to read, you need to go to the object to figure out why `this` and what it is for.